### ISSUE TITLE
Company profile enhancements

### ISSUE THREAD

**Issue by doomspork** [beam-community/elixir-companies#498]
Following my conversation with @Nitrino it was determined we need to create an issue it identify and implement some profile enhancements to companies. Some of these off the top of my head could include:

- Whether they consider themselves "remote" (despite having offices). I could see people searching on this.
- More complete support for social networks. Right now we store Github and a blog. We could and should allow for others: Twitter, Facebook, Medium, ... you get the idea. Can we find a way to support this without having to add new columns for each new type? I think we can and I think we can update the UI to handle this gracefully.

I'd love to get your thoughts @tajchumber since you do a lot of the UI work 😁

**Comment by Nitrino** [comment]
> Can we find a way to support this without having to add new columns for each new type?

Social networks usually do not require indexing and searching, so we can just use the PostgreSQL JSON array. e.g.: 
```json
[
  {
    "network": "twitter",
    "url": "https://twitter.com/elixirschool"
  },
  ...
]

**Comment by maartenvanvliet** [comment]
I'd like to tackle the "remote" option for companies. 
I suggest adding a remote true/false field to the companies table. Or are more options needed?

Also, what should the design look like?

**Comment by gemantzu** [comment]
@maartenvanvliet thank you for your help. I guess the more accurate solution would be to have a remote field in the jobs schema, because a company might have remote and non remote jobs to offer. Than, we could filter the remote jobs for anyone interested.

**Comment by tsrandrei** [comment]
One suggestion :  Logo of the company ? might help having a visual impact I think. Perhaps doing a redesign of cards might help?

**Comment by nathanl** [comment]
I've wanted an optional field on submissions for a link which shows that the company is using Elixir. It could be public or visible only to admins, but something like a blog post by one of their engineers or a job posting would substantiate the claim.

**Comment by maartenvanvliet** [comment]
Yes that would be helpful, I do some quick checks usually to check the claim (blogs, jobs, github repo's, phoenix html tags, cowboy http headers) but having submissions provide evidence would help 😅

**Comment by doomspork** [comment]
@dzachrich @LatitudeAdjustment @gemantzu @samuelpordeus I would love to revisit the conversation to enhance our company profiles further (they're admittedly a bit lackluster). Do any of you have thoughts? All ideas welcome 🎉

**Comment by gemantzu** [comment]
From the above, company logo should be a very good addition. The remote flag, I am still not sure about.

**Comment by doomspork** [comment]
Would take a bit of time to convert all the existing companies but thoughts on a format like this? 

```elixir
%{
  company: %{
    company_logo: "" # What sizes? 35x35?
    description: "A brief description of the company",
    github: "https://github.com/",
    industry: "IT Services",
    name: "Example",
    sector: "Information Technology",
    website: "https://www.example.com",
  },
  locations: [%{
    # Support N locations for companies spread out?
  }],
  elixir_usage: %{
    blog_posts: [
      # Links to blog posts on Elixir usage
    ],
    projects: [
      # Links to Open Source Elixir projects
    ],
    summary: "A summary of how Elixir is used",
  },
  last_changed_on: ~D[2023-03-11],
  remote: true
}
```

It contains all the current information with some additions:

- A section specific to Elixir usage with a summary, links to specific blog posts, and any open source Elixir projects
- Industry + Sector to better align with GICS 
- Support for multiple office locations
- A `remote` boolean
- Support for `logo`

**Comment by hassan** [comment]
This might somewhat overlap with "Elixir usage" but I would love to see an "adoption story" section describing how and why the company chose Elixir - something to show the CIO/CTO crowd that the decision was based on a rational risk/benefit analysis.

**Comment by doomspork** [comment]
@hassan this is a _great_ suggestion! I think this would go a very long way to helping the executive stakeholders as you mentioned. Thanks so much for sharing 🙏

**Comment by yordis** [comment]
Instead of `remote: true` maybe it is better to make it an enum?

```typescript
type BetterNameField = 'REMOTE' | 'ONSITE' | 'HYBRID'
```

**Comment by doomspork** [comment]
@yordis that's a great idea!
